.github/workflows/testnet.yml

```
@@ -0,0 +1,233 @@
+        required: false
+        default: 'helium-testnet'
+        type: string
+
+env:
+  CHAIN_ID: ${{ github.event.inputs.chain_id }}
+
+jobs:
+  deploy-testnet:
+    name: Deploy ${{ github.event.inputs.network_type }} node testnet
+    runs-on: ubuntu-latest
+    timeout-minutes: ${{ fromJSON(github.event.inputs.duration) + 30 }}
```

actionlint (1.7.7)

got unexpected character '+' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)